Documentation/project-docs/standard-platform.md

```diff
@@ -23,12 +23,14 @@ The .NET Platform Standard version represents binary portability across platform
 
 
 ## Terms
+- **.NET Platform Standard** - versioned sets of the reference assemblies that all .NET Platforms must support.
 - **PCL** - Portable Class Library
 - **Platform** - e.g. .NET Framework 4.5, Windows Phone 8.1
 - **Reference Assembly** - An assembly that contains API surface only. There is no IL in the method bodies. It is used for compilation only, and cannot be used to run. Also commonly referred to as "Contracts".
-- **Implementation Assembly** - An assembly that contains an implementation of a reference assembly. These are usually implemented in the platform itself and cannot be updated without updating the platform.
-- **.NET Platform Standard** - versioned sets of the reference assemblies.
-- **Multi-targeting** - to compile the same source code files to different target platforms, i.e. against different API sets.
+- **Implementation Assembly** - An assembly that contains an implementation of a reference assembly. These can be implemented as standalone assemblies but can sometimes be anchored by a platform and cannot be updated without updating the platform.
+- **Anchored Assembly** - An assembly that is not in the Platform Standard and is able to ship independently on top of many platforms but for specific platforms, the platform must be updated to support them. This set varies by platform.
+- **Multi-targeting** - to compile the same source code files to different target platforms, i.e. against different API sets
+- **Standard Library** - A blessed set of core .NET API's and versions that are prescribed to be used and supported together. This includes API's in the Platform Standard plus additional libraries that are core to .NET but built on top of the Platform Standard.
```

Comment:
> includes API's

Includes *all* of the APIs, right?

> plus additional libraries

So there's Platform Standard which provides a list A, some Standard Library that provides a list B which contains all of A, and a platform can support one or more versions of Platform Standard and one or more Standard Libraries?